src/app/components/utility/input/input.component.scss

```diff
@@ -130,6 +130,7 @@
     &:focus {
       outline: none;
       border-bottom: 1px solid $yellow;
+      box-shadow:none;
     }
```

Comment:
@nsjcorps Where is the colour changed?

Reply:
The color is defined in the $yellow variable as #ffaf4b.

Reply:
It is defined in the _variables.scss file as #ffaf4b.
![Screenshot (120)](https://user-images.githubusercontent.com/35180217/70053728-53b86700-15d6-11ea-9c5b-ce0d21e27815.png)